<!-- -*- mode: XML -*- -->
<launch>

    <param name="speed" type="double" value="0.3" />
    <param name="steering_gain" type="double" value="3" />
    <param name="steering_dgain" type="double" value="1" />
    <param name="steering_bias" type="double" value="0.0" />
        
    <node pkg="imredd_pkg" type="FollowRoad_node.py" name="follow_road" output="screen"/>
</launch>
        
